server/src/termdb.violin.js

```diff
@@ -137,7 +137,7 @@ function divideValues(q, data, tw) {
 		if (useLog === 'log') {
 			if (min === 0) min = Math.max(min, value.value)
 		}
-
+		const sampleType = data.sampleType.plural_name
```

Comment:
sampleType is missing in[ other ds](http://localhost:3000/?noheader=1&mass={%22dslabel%22:%22SJLife%22,%22genome%22:%22hg38%22,%22plots%22:[{%22chartType%22:%22summary%22,%22term%22:{%22id%22:%22hrtavg%22,%22q%22:{%22mode%22:%22continuous%22}}}],%22nav%22:{%22activeTab%22:1}})